app/models/odk/response_saver.rb

```diff
@@ -3,22 +3,30 @@
 module ODK
   # Saves responses and is responsible for handling serialization and duplicate errors
   class ResponseSaver
-    MAX_TRIES = 10
-    # For testing race conditions, stubbed in tests
-    SLEEP_TIMER = 0
+    # The more tries, the more likely it is to eventually succeed
+    MAX_TRIES = 12
```

Comment:
i trust your assessment here, but wondering how you arrived at 12?

Reply:
I just increased it by 20% up from 10; didn't want to add too many in case it causes servers to process requests more slowly now with the new delay (but at most 12 tries with up to 100 ms in between adds about 1s to request times in the worst case, so there's room to increase this if it helps significantly).

Reply:
gotcha!